src/services/cli-http-client.js

```diff
@@ -80,20 +89,20 @@ class CliRequestClient {
     try {
       const response = await this.http(options);
 
-      this.logger.debug('response.statusCode: ' + response.statusCode);
+      this.logger.debug('response.statusCode: ' + response.status);
       this.logger.debug('response.headers: ' + JSON.stringify(response.headers));
 
-      if (response.statusCode < 200 || response.statusCode >= 300) {
-        const parsed = JSON.parse(response.body);
-        throw new TwilioCliError(`Error code ${parsed.code} from Twilio: ${parsed.message}. See ${parsed.more_info} for more info.`, parsed.code);
-      }
-
       return response;
     } catch (error) {
       if (NETWORK_ERROR_CODES.has(error.code)) {
         throw new TwilioCliError(NETWORK_ERROR);
       }
 
+      if (error.response.status < 200 || error.response.status >= 300) {
```

Comment:
Since axios throws for non 2XXs, the status code and headers won't get logged like they were before above. Need to refactor to log the response if this is the case.

Reply:
I can change what status codes it throws for so it returns normally for non 2XXs